ddtrace/tracer/span.go

```diff
@@ -135,6 +150,35 @@ func (s *span) setTagError(value interface{}, debugStack bool) {
 	}
 }
 
+// takeStacktrace takes stacktrace
+func takeStacktrace(n, offset uint) string {
+	var builder strings.Builder
+	pcs := make([]uintptr, n)
+
+	i := 0
+	// +2 to exclude runtime.Callers and takeStacktrace
+	numFrames := runtime.Callers(2+int(offset), pcs)
+	frames := runtime.CallersFrames(pcs[:numFrames])
+
+	for {
```

Comment:
You can remove `i` up there and just have:
```go
for i := 0;; i++ {
   // ...
}
```

Reply:
Done!